website/docs/d/user_external_identity.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Can you please link to this doc in the `github.erb` file so it's discoverable?

Reply:
done [here](https://github.com/integrations/terraform-provider-github/pull/1796/commits/510eab872fb346e36f341e3459f5715d9e92b52e)!